src/main/java/org/jenkinsci/plugins/workflow/flow/FlowExecutionOwner.java

```diff
@@ -109,6 +111,14 @@ public String getUrlOfExecution() throws IOException {
     public abstract int hashCode();
 
     /**
+     * Gets a listener to which we may print general messages.
+     * Normally {@link StepContext#get} should be used, but in some cases there is no associated step.
+     */
```

Comment:
Generally have not bothered with `@since` on these plugin APIs. If you are already depending on the version that has it, there is little reason to care when it was introduced. If you are not, then you will not know it exists, unless you are browsing sources on GitHub or something. When in doubt, use the latest version.